app/components/provider_interface/condition_status_bulk_edit_form_component.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_with(
+  model: form_object,
+  url: provider_interface_application_choice_confirm_update_conditions_path(application_choice.id),
+  method: :patch,
+) do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <h1 class="govuk-heading-xl">
+    Confirm if the candidate has met all conditions
+  </h1>
+
+  <%= render SummaryListComponent.new(rows: [
```

Comment:
Can we generate this hash in the component's ruby file.

Reply:
👍  Also addressed in the other component, thanks!